.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,16 @@
+name: Deploy documentation website
+
+on:
+  # Runs on pushes targeting the `main` branch
+  push:
+    branches: [main]
+    paths:
+      - "docs/**"
+      - ".github/workflows/**" # define the concrete paths on which a change triggers this action, e.g. backend/**
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Why do we need workflow_dispatch when docs get automatically updated on main push. Where is the effective difference?

Reply:
```suggestion
```

irgendwer wollte einmal was manuell ausrollen. Dann machen wir es raus